### DISCUSSION TITLE
Implementation of Fly-Pie in other Linux desktop environments

### DISCUSSION THREAD

**Discussion by D3vil0p3r** [Schneegans/Fly-Pie#227] [Ideas & Feature Discussions]
Hello!
As all of you know, Fly-Pie is based on GNOME as desktop environment.
Will be in the future the chance to implement it also on environments as KDE, Cinnamon, MATE, Unity, Xfce, LXQt, and so on?

What could be the effort on porting the application to these environments?

Thanks!

**Comment by Schneegans** [comment]
This has been discussed quite frequently (#22, #118, #198). If you want to contribute to the discussion, please use the pinned issue #22. I'll post a note to #22 regarding the estimated effort!

**Comment by Schneegans** [comment]
I just published some [news regarding a platform-agnostic pie menu](https://ko-fi.com/post/Introducing-Ken-Do-L3L7L0FQ2) I am working on. So maybe this will become a reality after all :smile:
